webhooks/issue.py

```diff
@@ -127,6 +128,19 @@ def parse_issue_text(text):
 ```
 {logs}
 ```
+'''
+        import configparser
+        aur_item = '* [{}](https://aur.archlinux.org/packages/{})'
+        nvconfig = configparser.ConfigParser(dict_type=dict, allow_no_value=True)
+        nvconfig.read_file(open(config.REPODIR + '/nvchecker.ini', 'r'))
+        aur_pkgs = filter(lambda r: r[1] is not None,
+                          ((pkg, nvconfig.get(pkg, 'aur', fallback=None)) for pkg in packages))
+        aur_info = '\n'.join((aur_item.format(pkg, aur or pkg) for pkg, aur in aur_pkgs))
```

Comment:
Don't use `filter`. Use comprehension instead.

Also, we should refactor this into a standalone function.

Reply:
It will be `aur_pkgs = (r for r in ((pkg, nvconfig.get(pkg, 'aur', fallback=None)) for pkg in packages) if r[1])` here if using comprehension, is it okay?

Reply:
Is there any way we can simplify the code but use `nvconfig.get()` no more than one time?

Reply:
呃，那还是这样吧。（想起了 [PEP 572](https://www.python.org/dev/peps/pep-0572/)）

Reply:
嗯，感觉 PEP 572 很有必要……